recommender/app/lib/Recommender.scala

```diff
@@ -21,12 +21,22 @@ class Recommender(esClient: TransportClient, userHistoryClient: UserHistoryClien
   val esType = "items"
   val popRank = "popRank"
   val popRankType = "double"
+  val historySize = 50
 
-  def getRecommendations(id: String, dateFilter: Option[DateRangeFilter], pageSize: Int): Future[List[RecommendationItems]] =
-    getRecommendations(id, dateFilter, pageSize, offset = 0)
+  def getRecommendationsForBrowserId(browserId: String, dateFilter: Option[DateRangeFilter], pageSize: Int): Future[List[RecommendationItems]] =
+    getRecommendationsForBrowserId(browserId, dateFilter, pageSize, offset = 0)
 
-  def getRecommendations(id: String, dateFilter: Option[DateRangeFilter], pageSize: Int, offset: Int): Future[List[RecommendationItems]] = {
-    userHistoryClient.articlesForBrowser(id) flatMap { articleIds =>
+  def getRecommendationsForBrowserId(browserId: String, dateFilter: Option[DateRangeFilter], pageSize: Int, offset: Int): Future[List[RecommendationItems]] = {
```

Comment:
why not to make offset param with default value = 0 ?
Have you considered introducing parameter object here, number of arguments of this methods will grow indefinitely otherwise